server-phoenix/lib/helios/locations.ex

```diff
@@ -0,0 +1,167 @@
+defmodule Helios.Locations do
```

Comment:
We also need to change the alias for Location in `seeds.exs` so `mix setup` doesn't error out

Reply:
yup thats one of the errors I just found. Gotta do a better job looking through all these imports lol.